src/test/java/pages/BasePage.java

```diff
@@ -15,31 +17,46 @@ public class BasePage {
     public WebDriver driver;
     public WebDriverWait wait;
     public Actions actions;
+    public String playlistName = null;
+    
+    //Web Element
+    @FindBy (xpath="//a[contains(text(), 'All Songs')]")
+    WebElement allSongsButton;
+    @FindBy (css=".playlist:nth-child(3)")
+    WebElement firstPlaylist;
```

Comment:
Base Page \ Base Class — Implements the necessary methods for work with webdriver.

We will store the web elements present on each page in the corresponding java class.

The webelements  allSongsButton, firstPlaylist;  must be stored in the `class HomePage`  or `class CustomPlaylistPage` because the playlist is visible on the web site -home page

![image](https://user-images.githubusercontent.com/104761482/213643559-1039d706-ee09-4e6b-8ff0-a92bd0df401b.png)